web/js/jquery.gerbera.editor.js

```diff
@@ -110,6 +110,47 @@
     modal.find('#editState').val('').prop('disabled', false);
     modal.find('#editProtocol').val('').prop('disabled', false);
     modal.find('#editSave').text('Save Item');
+
+    modal.find('#metadata').empty();
+    modal.find('#auxdata').empty();
+    modal.find('#resdata').empty();
```

Comment:
I'm not an expert in html or js but I hope modern web technologies can do better than such logic scattered through multiple methods and html tags in strings...
Something like templates and bindings xaml in c# had for ages.

Reply:
That's the way it works with jquery which is rather lowlevel. If we had stuff like Angular running it would look different.